### PR TITLE
Independent Publisher Worldwide Holidays 

### PR DESCRIPTION
**Pull request by reshmee011**
- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
image

- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
image

- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: independent-publisher-connectors/Wordwide Holidays/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CountryInfoDto"
+            }
+          }
+        },
+        "description": "Get country info for the given country",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Wordwide Holidays/apiDefinition.swagger.json:46]: @reshmee011 is it possible to give a different description than the summary? please fix all descriptions in the swagger file.

File: independent-publisher-connectors/Wordwide Holidays/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+        "description": "Get public holidays"
+      }
+    },
+    "/api/v3/IsTodayPublicHoliday/{countryCode}": {
+      "get": {
+        "tags": [
+          "PublicHoliday"
+        ],
+        "summary": "Is today a public holiday",
+        "description": "The calculation is made on the basis of UTC time to adjust the time please use the offset.<br />\r\nThis is a special endpoint for `curl`<br /><br />\r\n200 = Today is a public holiday<br />\r\n204 = Today is not a public holiday<br /><br />\r\n`STATUSCODE=$(curl --silent --output /dev/stderr --write-out \"%{http_code}\" https://date.nager.at/Api/v2/IsTodayPublicHoliday/AT)`<br /><br />\r\n`if [ $STATUSCODE -ne 200 ]; then # error handling; fi`",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Wordwide Holidays/apiDefinition.swagger.json:180]: @reshmee011 all descriptions should be in a good sentence format and I would suggest you to keep it short to fit in the text box. Please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]
@reshmee011 Thank you so much for submitting your new connector with us. May I ask you please to add "(Independent Publisher)" to your PR title?
Thank you so much again.

**Comment by reshmee011** [comment]
@Amjed-Ayoub : Thanks for the awesome review. I have resolved the review comments. Let me know if there are more to update.

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @reshmee011,
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by reshmee011** [comment]
@Amjed-Ayoub : Amazing , really pleased that the process was quite sleek:)

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]
Hello @reshmee011,

I hope you are doing well. we found out that the folder name is "Wordwide Holidays" and the connector name is "Worldwide Holidays". can you please fix the folder name.
image

Please accept me apologies for not catching this from the beginning.

Thank you so much again for you hard work with us.

**Comment by reshmee011** [comment]
Hi @Amjed-Ayoub : likewise I should have spotted it too as well... Thanks very much for spotting it. I have renamed the folder to the correct name.

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @reshmee011,
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by Amjed-Ayoub** [comment]
Hello @reshmee011,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by reshmee011** [comment]
@Amjed-Ayoub : Apologies , the PR got closed automatically while getting latest on my branch. Let me if it caused any issues.

**Review by Amjed-Ayoub — APPROVED** [review]
Hello @reshmee011,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program.
Thank you very much for working with us.

**Comment by reshmee011** [comment]
@Amjed-Ayoub : Thank you very much for merging the PR, looking forward to see it live.

**Comment by Amjed-Ayoub** [comment]
@reshmee011 dont worry, your connector is deployed and this PR is merged.

**Pull request merged**